docs/guide.rst

```diff
@@ -878,7 +878,7 @@ copied to the guest.
 Conditional step configuration
 ------------------------------
 
-.. versionadded:: 1.39
+.. versionadded:: 1.40
```

Comment:
Shouldn't these be 1.40.0 as well? Or would it break something? (not blocking release PR, just noticed it here)

Reply:
Hm, good question. We're not adding new features in micro versions so I'd say it's ok. But if we want to use `x.y.z` versions across the whole documentation consistently, let's do that in a separate pull request and cover all occurences.